boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -289,7 +289,7 @@
         const struct device* wdt =                            \
             DEVICE_DT_GET(DT_ALIAS(watchdog0));               \
         if (device_is_ready(wdt)) {                           \
-                wdt_feed(wtd, 0);                             \
+                wdt_feed(wdt, 0);                             \
```

Comment:
This is fixing only one of the errors, there is another typo below which is older and needs fixing as well. Also please cleanup the commit message, there are typos.

I will take a look at the CI later today and fix it to catch errors like this, before this PR goes in...

Reply:
Uhhh, I've created the patches using a different repository than the one I was testing the changes on and made a typo. There is another one: https://github.com/mcu-tools/mcuboot/blob/0ee9bcaeab2a000f13d14745ce0fce8cb8986d2b/boot/zephyr/include/mcuboot_config/mcuboot_config.h#L279. It would be nice to have more boards in CI to catch such mistakes.